pages/docs/components/layout/grid.mdx

```diff
@@ -75,6 +75,29 @@ start or end at the `nth` grid position.
 </Grid>
 ```
 
+## Template areas
+
+The `templateAreas` prop specifies areas within the grid layout. Use template
+literals to name the area. Now you can reference the area by passing `area` prop
+in the `<GridItem />` component.
+
+```jsx
+<Grid
+  templateAreas={`"header header"
+                  "nav main"
+                  "nav footer"`}
+  gridTemplateRows={'50px 1fr 30px'}
+  gridTemplateColumns={'150px 1fr'}
+  h='200px'
+  gap={1}
+>
+  <GridItem bg='orange.300' area={'header'} />
+  <GridItem bg='pink.300' area={'nav'} />
+  <GridItem bg='green.300' area={'main'} />
+  <GridItem bg='blue.300' area={'footer'} />
```

Comment:
I think it's good to leave the text. To make it more visible, you can play with the font size and weight.

Reply:
Working on it 💻